modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml

```diff
@@ -6460,9 +6460,41 @@ paths:
                                 type: array
             tags: ["ContentTemplate"]
     "/asset-libraries/{assetLibraryId}/document-data-definition-types":
+        get:
```

Comment:
Does it make sense to have also page and pageSize?

Reply:
Done